<launch>
<node pkg="tf" type="static_transform_publisher" name="world_to_cam_0"
    args="0 0 0 0 0 0 world usb_cam 1 00" />
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam_0" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" value="/home/laboratorio/ros_workspace/src/proyecto_final/data/camera_data/ost.yaml"/>
  </node>
  <node name="image_view_0" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam_0/image_raw"/>
    <param name="autosize" value="true" />
  </node>

  <node pkg="tf" type="static_transform_publisher" name="world_to_cam_1"
    args="0 0 0 0 0 0 world usb_cam 1 00" />
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam_1" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" value="/home/laboratorio/ros_workspace/src/proyecto_final/data/camera_data/ost.yaml"/>
  </node>
  <node name="image_view_1" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam_1/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>


<!-- image_proc -->
